src/test/scala/nl.knaw.dans.easy.sword2/ReadmeSpec.scala

```diff
@@ -44,6 +44,12 @@ class ReadmeSpec extends TestSupportFixture with CustomMatchers {
     setProperty("cleanup.INVALID", "no")
     setProperty("cleanup.REJECTED", "no")
     setProperty("cleanup.FAILED", "no")
+    setProperty("easy-deposit-properties.url", "http://localhost:20200/")
+    setProperty("easy-deposit-properties.username", "easy-deposit-properties")
+    setProperty("easy-deposit-properties.password", "changeme")
+    setProperty("easy-deposit-properties.conn-timeout-ms", 5000)
+    setProperty("easy-deposit-properties.read-timeout-ms", 5000)
+    setProperty("easy-deposit-properties.mode", "FILE")
```

Comment:
Can't you load the debug properties? Perhaps just change one or two values.

Reply:
great idea! consider it done!